api/src/types/transformed/articles.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  TransformedArticleFormat,
+  TransformedContributor,
+  TransformedImage,
+} from "..";
+
+export type TransformedArticle = {
```

Comment:
Can we call this an `Article` as it's a first class type in our world? And ditto for other `Transformed*` types - a way of putting it could be that they aren't `Articles` that have been `Transformed`, they're just `Articles`.